includes/Story_Renderer/HTML.php

```diff
@@ -205,6 +205,10 @@ protected function replace_html_head( $content ) {
 		$start_tag = '<meta name="web-stories-replace-head-start"/>';
 		$end_tag   = '<meta name="web-stories-replace-head-end"/>';
 
+		// Replace malformed meta tags with correct tags.
+		$content = (string) preg_replace( '/<meta name="web-stories-replace-head-start\s?"\s?\/>/i', $start_tag, $content );
+		$content = (string) preg_replace( '/<meta name="web-stories-replace-head-end\s?"\s?\/>/i', $end_tag, $content );
```

Comment:
Don't need to check for whitespace within the attribute value.

```suggestion
		$content = (string) preg_replace( '/<meta name="web-stories-replace-head-start"\s?\/>/i', $start_tag, $content );
		$content = (string) preg_replace( '/<meta name="web-stories-replace-head-end"\s?\/>/i', $end_tag, $content );
```

Reply:
> Not sure yet why (just React doing its thing I guess), but it looks like this:

`<meta name="web-stories-replace-head-start "/>`
`<meta name="web-stories-replace-head-end" />`

From [here](https://github.com/google/web-stories-wp/issues/5589#issue-758695705)

This isn't a mistake, this is from the description of the ticket.

Reply:
My apologies, I think I simply made a typo in the ticket.

Since it doesn't really hurt we can also just leave this as-is.

Reply:
I agree, leave it in, more error handling never heart anyway